rhel-minimal/Dockerfile

```diff
@@ -34,7 +34,8 @@ ENV RHEL_VERSION $RHEL_VERSION
 RUN set -ex; \
     if [ "$ASSET" = "ce" ] ; then \
       curl -fL "https://download.konghq.com/gateway-${KONG_VERSION%%.*}.x-rhel-${RHEL_VERSION}/Packages/k/kong-${KONG_VERSION}.rhel${RHEL_VERSION}.amd64.rpm" -o /tmp/kong.rpm \
-      && echo "$KONG_SHA256  /tmp/kong.rpm" | sha256sum -c -; \
+      && echo "$KONG_SHA256  /tmp/kong.rpm" | sha256sum -c - \
+      || exit 1; \
```

Comment:
is the `|| exit 1;` necessary?

Reply:
I think so. I was noticing cases where the asset == `ce`, the sha was incorrect, but the build wasn't failing until `kong version` failed to exist.